app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ActiveRecord::Base
+	has_many :games
```

Comment:
This won't work because a user doesn't have many games directly.  A user actually has many white games and black games.  Thus you have to specify games as a combination of white and black games.  There are several techniques for doing this and we can discuss on our call.   That said, the way to catch these things and validate that everything is working how you expect is to write unit tests that verify these relationships...